homeassistant/components/osoenergy/strings.json

```diff
@@ -25,6 +25,17 @@
     }
   },
   "entity": {
+    "binary_sensor": {
+      "power_save": {
+        "name": "Power save"
+      },
+      "extra_energy": {
+        "name": "Extra energy"
+      },
+      "heater_state": {
+        "name": "Heater state"
```

Comment:
Can we have a better name for it? Try to avoid `state` in the name as it sounds strange: "the state of the Heater state is on"...